superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -93,6 +94,7 @@ class SqlEditor extends React.PureComponent {
       northPercent: props.queryEditor.northPercent || INITIAL_NORTH_PERCENT,
       southPercent: props.queryEditor.southPercent || INITIAL_SOUTH_PERCENT,
       sql: props.queryEditor.sql,
+      autocompleteEnabled: true,
```

Comment:
Not blocking, but i'll bet once we roll this out, people will want the ability to default this setting or to have it grab the setting from the current tab when creating the new one.

Food for thought

Reply:
good point. We'll see how popular disabling is. It could be as @mistercrunch suspects that most wouldn't want it disabled.